README.md

```diff
@@ -25,6 +25,8 @@ F3D also contains the libf3d, a simple library to render meshes, with C++, Pytho
 
 See the [gallery](doc/GALLERY.md) for more images, take a look at the [changelog](doc/CHANGELOG.md) or go to the [releases page](https://github.com/f3d-app/f3d/releases) to download F3D!
 
+If you need any help or want to discuss with other F3D users and developers, head over to our [discord](https://discord.gg/ZYHzNeEdX5).
```

Comment:
Can we add a badge on the homepage too?

Reply:
Is the link valid permanently?

Reply:
Good point for the badge !

Yes, link is never expiring.

Reply:
![a](https://user-images.githubusercontent.com/3129530/204085079-58e1794e-f937-4301-8c56-2a1c23be6f87.png)